### PR TITLE
Csv import/support postgres

### PR DESCRIPTION
**Pull request by sloypog**
### CATEGORY

Choose one

- [ ] Bug Fix
- [X] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
Add support for importing a csv-file into a new postgres database

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/views/csv_import.py [hunk 1]
```diff
@@ -101,7 +106,7 @@ def add(self) -> Response:
 
         try:
             if database_id != -1:
-                schema = None if not form_data["schema"] else form_data["schema"]
+                schema = form_data["schema"] or None
```
Comment (michellekunz) [inline superset/views/csv_import.py:109]: Why did you remove the if?

File: superset/views/csv_import.py [hunk 2]
```diff
@@ -168,56 +179,82 @@ def add(self) -> Response:
...
                 db.session.commit()
-            except OSError:
-                message = _(
-                    "Error when trying to create Database.The database file {0}.db could not be removed. "
-                    "Please contact your administrator to remove it manually".format(
+                return item
+            except Exception as e:
+                exception = e
+                if isinstance(e, IntegrityError):
+                    message = "Error when trying to create Database. A database with the name {0} already exists.".format(
```
Comment (michellekunz) [inline superset/views/csv_import.py]: Theoretically, IntegrityError could stand for a lot, not only for database name already exists. Maybe a more generic error message would be better

**Review by michellekunz — CHANGES_REQUESTED** [review]

File: superset/views/csv_import.py (hunk 1 above)
Comment (sloypog) [reply superset/views/csv_import.py:109]: was proposed in the PR into superset-master, therefore I changed it here too (https://github.com/apache/incubator-superset/pull/8556/files/2ca3740d13850b18116ace1ce8c0990885ff5280#r346021226)

File: superset/views/csv_import.py (hunk 2 above)
Comment (sloypog) [reply superset/views/csv_import.py]: This is code from the master, but makes sense, changed it nevertheless

**Review by michellekunz — APPROVED** [review]
